example/lib/main.dart

```diff
@@ -43,7 +44,21 @@ class _MyAppState extends State<MyApp> {
   }
 
   Future scan() async {
-    String barcode = await BarcodeScanner.scan();
-    setState(() => this.barcode = barcode);
+    try {
+      String barcode = await BarcodeScanner.scan();
+      setState(() => this.barcode = barcode);
+    } on PlatformException catch (e) {
+      if (e.code == 'PERMISSION_NOT_GRANTED') {
```

Comment:
Lets turn `PERMISSION_NOT_GRANTED` into a constant so we can use `if (e.code == BarcodeScanner.CameraAccessDenied` or something similar.

Reply:
Good idea!